tools/docker/Dockerfile.jinja

```diff
@@ -16,10 +16,18 @@ RUN apt update && apt-get install -y --no-install-recommends \
     git-core \ 
     gnupg \ 
     libssl-dev \ 
+    jq \
     python3-pip \ 
     python-is-python3 \ 
     wget
 
+#= Download dependency =====================================================
+
+WORKDIR /root/dependency
+RUN wget https://raw.githubusercontent.com/asterinas/linux_vdso/2a6d2db/vdso64.so -O vdso64.so
+RUN wget https://raw.githubusercontent.com/asterinas/linux_kernel/9e66d28/vmlinuz-5.15.0-105-generic \
+        -O vmlinuz
```

Comment:
Extract the modifications to the Dockerfile into another PR and bump the version to upload the new Docker image

Reply:
Now #978 bumps the version.

Reply:
#978 has been merged. Now remove the changes to the Dockerfile in this PR. And rebase to the latest main branch.